sass/_statistic.scss

```diff
@@ -0,0 +1,51 @@
+
+.statis{
```

Comment:
You can probably nest all your styles inside the statis-class. That way it's easier to avoid conflict with other peoples styles.